packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -438,10 +439,16 @@ export class ServerlessCluster extends ServerlessClusterBase {
       }),
     ];
 
+    let clusterIdentifier = props.clusterIdentifier;
+    const clusterIdentifierToLowercaseFeatureEnabled = FeatureFlags.of(this).isEnabled(cxapi.RDS_CLUSTER_IDENTIFIER_TO_LOWERCASE);
+    if ( clusterIdentifierToLowercaseFeatureEnabled ) {
+      clusterIdentifier = clusterIdentifier?.toLowerCase();
+    }
```

Comment:
Let's rewrite this to a ternary operator. No need to make things mutable when it's not needed.

```suggestion
    const clusterIdentifier = FeatureFlags.of(this).isEnabled(cxapi.RDS_CLUSTER_IDENTIFIER_TO_LOWERCASE)
      ? props.clusterIdentifier?.toLowerCase()
      : props.clusterIdentifier;
```